app/src/main/java/com/codesoom/assignment/errors/InvalidPasswordException.java

```diff
@@ -0,0 +1,10 @@
+package com.codesoom.assignment.errors;
+
+/**
+ * 일치하지 않는 비밀번호일 때 예외를 던진다.
+ */
```

Comment:
이렇게 할 수도 있겠죠.
```suggestion
/**
 * 비밀번호가 일치하지 않을 때 던집니다.
 */
```

이런 식의 설명은 Java의 빌트인 예외에 달려있는 JavaDoc을 읽다 보시면 자주 발견하실 수 있을 거에요.
![image](https://user-images.githubusercontent.com/1855714/166105831-7953d50d-6134-4316-8a57-df9ef615b65a.png)